src/components/footer/Footer.tsx

```diff
@@ -27,6 +25,7 @@ export function Footer(props: FooterProps) {
 	const {
 		brandTop,
 		accessibility,
+		accessibilityLinkProps,
 		homeLinkProps,
 		contentDescription,
 		personalDataLinkProps,
```

Comment:
I will comment out the `bottomItems` prop as it's not used currently, again to see if it resolves the gh action problem.